tests/RandomComplexTest.php

```diff
@@ -86,6 +86,55 @@ public function testCreatingMultipleComplexWithNegativeRealsHavingSameWholePart(
         }
     }
 
+    public function testGetTheCorrectName()
+    {
+        $r = new RandomComplex();
+        $this->assertEquals('r', $r->__get('r'));
```

Comment:
No, sorry, but this breaks original behavior. Please see https://github.com/malenkiki/math/pull/25/commits/fa42d3500c88374787a532d35f0fcabf8242250e#r150422922